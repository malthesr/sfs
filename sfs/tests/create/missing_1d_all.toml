args = [
  "create",
  "--allow-stdin",
  "-v",
  "tests/create/missing.bcf"
]
