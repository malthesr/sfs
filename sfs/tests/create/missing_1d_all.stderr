[sfs  info] Skipping site 'chr1:1' due to too many missing and/or multiallelic genotypes. This message will be shown only once, with a summary at the end. Increase verbosity for more information.
[sfs debug] Skipping sample 'sample0' at site 'chr1:1'. Reason: 'missing'.
[sfs debug] Skipping sample 'sample3' at site 'chr1:2'. Reason: 'missing'.
[sfs debug] Skipping sample 'sample4' at site 'chr1:2'. Reason: 'missing'.
[sfs debug] Skipping sample 'sample2' at site 'chr1:3'. Reason: 'missing'.
[sfs debug] Skipping sample 'sample3' at site 'chr1:4'. Reason: 'missing'.
[sfs  info] Skipped 4/5 sites due to missing and/or multiallelic genotypes. Project data (or relax projection) as necessary to keep more sites.
